inc/options/browsercache.php

```diff
@@ -1,449 +1,652 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
+}
 
 $security_session_values = array(
-	'' => 'Default',
-	'on' => 'Enable',
-	'off' => 'Disable'
+	''    => 'Default',
+	'on'  => 'Enable',
+	'off' => 'Disable',
 );
 
 ?>
-<?php include W3TC_INC_DIR . '/options/common/header.php'; ?>
+<?php require W3TC_INC_DIR . '/options/common/header.php'; ?>
 
-<form action="admin.php?page=<?php echo $this->_page; ?>" method="post">
+<form action="admin.php?page=<?php echo esc_attr( $this->_page ); ?>" method="post">
 	<p>
-		<?php echo sprintf( __( 'Browser caching is currently %s.', 'w3-total-cache' ), '<span class="w3tc-' . ( $browsercache_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>' ) ?>
+		<?php
+		echo wp_kses(
+			sprintf(
+				// translators: 1 HTML span tag indicating Browsercache enabled/disabled.
+				__(
+					'Browser caching is currently %1$s.',
+					'w3-total-cache'
+				),
+				'<span class="w3tc-' . ( $browsercache_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>'
```

Comment:
Use `esc_html__()`.

Reply:
Changed to esc_html__ for both instances